tests/smppmock.py

```diff
@@ -35,7 +35,10 @@
 import inspect
 from collections import namedtuple, Sequence, Sized
 from functools import update_wrapper
-from smpplib.client import ConnectionError
+if six.PY2:
+    from smpplib.client import ConnectionError
+else:
+    from smpplib.exceptions import ConnectionError
```

Comment:
Just so I understand correctly: smpplib not work with Python 3, does it?

Reply:
Correct, it does not work with Python 3 (no changes since 2010).
There are some other smpp packages but none of them can be used directly.
The most promising seems to be <https://github.com/podshumok/python-smpplib>.

Reply:
:+1: